app/models/exercise.rb

```diff
@@ -10,6 +10,7 @@ class Exercise < ApplicationRecord
   include WithNumber,
           WithName,
           WithAssignments,
+          WithProgress,
```

Comment:
I think this isn't necessary now, because I recall you were adding it in order to use `completed_for?`
If it's in fact necessary, we should discuss another way of including it. Exercise doesn't meet the required interface to be a progress item, as it doesn't have structural_children, for example.

Reply:
Right, undoing this.